src/Chart/BarChart.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { symToStr } from "tsafe/symToStr";
+import "@gouvfr/dsfr-chart/BarChart/bar-chart.common";
+import "@gouvfr/dsfr-chart/BarChart/bar-chart.css";
+import {
+    chartWrapper,
+    IntrinsicGraphType,
+    BaseChartProps,
+    stringifyObjectValue,
+    MultiChartProps,
+    ChartLineProps,
+    IntrinsicGraphLineType
+} from "./chartWrapper";
+
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            // https://github.com/GouvernementFR/dsfr-chart/blob/v1.0.0/src/components/BarChart.vue#L79
+            "bar-chart": {
+                horizontal?: string;
+                stacked?: string;
+            } & IntrinsicGraphType &
+                IntrinsicGraphLineType;
+        }
+    }
+}
+
+export type BarChartBaseProps = {
+    horizontal?: boolean;
+    stacked?: boolean;
+} & MultiChartProps &
+    ChartLineProps;
+
+export type BarChartProps = BarChartBaseProps & BaseChartProps;
+
+/** @see <https://components.react-dsfr.codegouv.studio/?path=/docs/components-chart> */
+export const BarChart = chartWrapper((props: BarChartBaseProps) => {
+    return <bar-chart {...stringifyObjectValue(props)} />;
```

Comment:
This is probably not Server Renderable and thus it won't work in next.js projects.  

We'll have to instruct next user to use `dynamic()` https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#with-no-ssr

Reply:
Yes, maybe it will be the case. I will test it and add a guide/example if this is the case :)